api/CanvasPattern.json

```diff
@@ -72,10 +72,10 @@
               "version_added": false
             },
             "opera": {
-              "version_added": "9"
```

Comment:
Did you check that this wasn't in Presto? The update script bails with data like that (multiple ranges) so if it was it wouldn't make the right edit.

Reply:
Whoops, this change wasn't meant to be here -- reverted!